po/R-es.po

```diff
@@ -168,27 +168,27 @@ msgstr "Tutoriales"
 #: development.R:20
 #, fuzzy
 msgid "Released version"
-msgstr "Version official"
+msgstr "Versión lanzada"
 
 #: development.R:21
 msgid "In-development version"
 msgstr "Versión en desarrollo"
 
 #: development.R:22
 msgid "Unreleased version"
-msgstr "Versión no en CRAN"
+msgstr "Versión en desarrollo"
```

Comment:
Esta traducción quedó justo igual a la anterior. Quizás podría ser "Versión no lanzada" o "Versión no publicada"

Reply:
This message is relatively rarely used, but it is meant to indicate that it's a development version of a package that has never been released to CRAN (as opposed to development version that's newer than the version on CRAN).

Reply:
My issue here is that the same translation is proposed for "In-development version" and "Unreleased version"

Reply:
Oh! I didn’t realise, bien visto @rivaquiroga

Reply:
"Versión no publicada". Al final:

```

#: development.R:20
msgid "Released version"
msgstr "Versión publicada"

#: development.R:21
msgid "In-development version"
msgstr "Versión en desarrollo"

#: development.R:22
msgid "Unreleased version"
msgstr "Versión no publicada"

```